internal/pkg/externalplugins/owners/owners_test.go

```diff
@@ -141,9 +143,10 @@ func TestListOwners(t *testing.T) {
 		sigRes                 *SigResponse
 		labels                 []github.Label
 		useDefaultSigName      bool
-		trustTeam              string
+		trustTeams             []string
 		defaultRequireLgtm     int
 		requireLgtmLabelPrefix string
+		pluginConfig           tiexternalplugins.TiCommunityOwners
```

Comment:
I think we should keep the changes minimal and just add brachconfig in this place, instead of constructing all the configurations.